docs/using-tre/tre-for-research/importing-exporting-data.md

```diff
@@ -0,0 +1,95 @@
+
+### Step 3: Get your approved data
+
+The request will be in an *In Review* state until it's either approved or rejected by your Airlock Manager(s) manually or by an automated workflow, depending on your organisation's specific configuration.
+
+!!! note
+    Your organisation may have the Airlock Notifier service configured which will send email notifications when your request has been approved/rejected, or you may have another mechanism in place. Please check with your TRE administrator.
+
+If the request is rejected, your data will be deleted and your request will move into a *Rejected* state. You will be able to see feedback in the **Reviews** section on why your request was rejected so you can create a new request that addresses any concerns.
+
+If your request is approved, you can follow the below steps to get your data from within your workspace:
+
+1. Head back to your Airlock request in the TRE UI. You should find that it is now in an *Approved* state and ready for you to get your data. You can also see the notes from the reviewer in the **Reviews** section.
+
+[![Get download link](../../assets/get-request-download-link.png)](../../assets/get-request-download-link.png)
+
+2. Click *Generate* in the **Files** section to generate another Storage SAS URL which you'll use for downloading your data.
+
+3. Paste this link into your Workspace VM (or whichever workspace resource you're wanting to access the data from). Like before, use your preferred tool to access the data using the SAS URL, but this time to download the data.
+
+    - With the Azure CLI, you can use `az storage blob download --file /path/to/write/to --blob-url SAS_URL`. [More info](https://docs.microsoft.com/en-us/cli/azure/storage/blob?view=azure-cli-latest#az-storage-blob-download)
+
+!!! tip
+    If you're using a Workspace VM that uses one of the standard TRE Data Science VM images, you'll likely have both Storage Explorer and the Azure CLI pre-installed.
+
+## Exporting data from a workspace
+
+Exporting data from a secure TRE workspace to the outside world involves similar steps to Import, but with a couple of key differences. Follow these steps:
+
+1. Open your TRE UI and navigate to the workspace you wish to export data from
+
+2. Navigate to the Airlock tab (in the left-hand menu) and click *Create new*, then select *Export*
+
+3. Fill in a suitable **Title** and **Business Justification** for the request then hit *Create*
+
+4. Once the draft request pop-out opens, click *Generate* in the **Files** section to generate a Storage SAS URL to use for uploading your data.
+
+5. You now need to head into your Workspace VM/resource containing the data you wish to export, and paste in the SAS URL you've just generated. Use your preferred storage tool to upload the data to the request container. See Step 2 in the [Importing data](#importing-data-to-a-workspace) section for more details on examples of these tools
```

Comment:
nit: maybe "for more details on using these tools" or "for more examples of these tools"?